bench/micro/Bench/Database/LSMTree/Internal/Lookup.hs

```diff
@@ -78,39 +86,25 @@ benchmarks = bgroup "Bench.Database.LSMTree.Internal.Lookup" [
             bench "Bloomfilter query"    $ whnf (elems b) ks
             -- the compact index is only searched for (true and false) positive
             -- lookup keys
-          , env (pure $ filter (`Bloom.elem` b) ks) $ \ks' ->
-            bench "Compact index search" $ whnf (searches ci) ks'
+          , env (pure $ V.filter (`Bloom.elem` b) ks) $ \ks' ->
+              bench "Compact index search" $ whnf (searches ci) ks'
```

Comment:
I should rewrite these two benchmarks to target `bloomQueries` and `indexSearches` instead